content/docs/scale/_index.md

```diff
@@ -41,39 +42,66 @@ The test was run in different OSM form factors, factoring in different amounts o
 	- Correctness is ensured. It is checked that all TrafficSplit server members are eventually reached.
 - Test timeout for network correctness: 150 seconds
 
-## Assessment and Limits
+# Assessment and Limits
 *Note: Assuming proxy per pod, so pod/proxies can be used interchangeably.*
-### CPU
-#### OSM Controller
-- **1vcpu per 500 proxies**, however giving more cpu does not scale linearly (m<1) with current architecture; horizontal scaling should be considered to increase supported mesh size. 
-- Network settlment times vary from **<10s with no pods to +2min at 700 pods**. 
+
+Test failed at around 1200 pods, with kubernetes unable to bring up in time a pod in the mesh.
+## CPU
+### OSM Controller
+- **1vcpu per 700 proxies**, giving more cpu does not scale linearly (m<1) with current architecture; horizontal scaling should be considered to increase supported mesh size. 
+- Network settlment times vary from **<10s with no pods to +2min at 1000 pods**. 
 
 <p align="center">
-  <img src="../images/scale/cpu.png" width="550" height="275"/>
+  <img src="../images/scale/prox.png" width="750" height="120"/>
 </p>
-<center><i>In blue, OSM controller, orange for OSM Injector, red for Prometheus</i></center><br>
+<center><i>Fig 1. Function of number of proxies onboarded during the test. Test period is ~1h.</i></center><br>
 
-#### OSM Injector
-- Injector shows constant handling of 20 pods at a time, with constant times to create the 2048bit certificates and webhook handling staying around the 2-5sec (accounting for the certificate creation).
+<p align="center">
+  <img src="../images/scale/cpu.png" width="750" height="225"/>
+</p>
+<center><i>Fig 2. Function of CPU load; OSM in yellow, CPU time for each iteration (spike) increases in time per iteration.</i></center><br>
 
-#### Prometheus
-- Prometheus shows an increase of CPU per scrapped pod to almost **1vcpu per 700 pods**.
-This has to take into account our default settings and our scrapping options (which, for example, we use quite low `scrape_interval`), which heavily affect the resources consumed.
+#### ADS Performance
+- With the recent ADS pipelining changes in OSM, it is ensured not too many ADS updates are scheduled for busy work at the same time, ensuring low times as granted by the available CPU.
+This yields more deterministic results, with all updates always under sub 0.10s window, and serialization of number of events as opposed to arbitrary scheduling from Golang.
+<p align="center">
+  <img src="../images/scale/histogram.png" width="1250" height="225"/>
+</p>
+<center><i>Fig 3. Histogram of ADS computation timings. (All verticals) </i></center><br>
+
+- The number of XDS updates over the test grows additively with any current number of onboarded proxies, each iteration occupying more time until basically iterations overlap, given the rate at which OSM can compute ADS updates:
+<p align="center">
+  <img src="../images/scale/ads.png" width="1250" height="225"/>
```

Comment:
This image doesn't seem to be rendering for some reason.

This is what I see here: https://github.com/openservicemesh/osm-docs/blob/d4f4da67c2901e1fd6737927cd2d855bb72220c1/content/docs/scale/_index.md#ads-performance

![image](https://user-images.githubusercontent.com/16093815/118292965-c8b3ef80-b49e-11eb-9069-e547e6a2cf5d.png)

Reply:
if you reference images as `/docs/images/scale/<whatever>` they will now work correctly on the website and in preview on netlify.

Reply:
I saw it doesn't render, but I thought I saw it render on the page (incorrectly). But the image seems to be valid in the branch. Not quite sure what is going on.

https://raw.githubusercontent.com/openservicemesh/osm-docs/d4f4da67c2901e1fd6737927cd2d855bb72220c1/content/docs/images/scale/ads.png

Any thoughts?

Reply:
I have removed the image and reference for now. I might reintroduce them at a later time.

Reply:
Actually looks fine to me now. And I now realize I should've been the website view, but I'm surprised that's the only one that wasn't working.

Reply:
Oh nvm, I was looking at the wrong image. I see you just removed this one.